packages/drivers/odsp-urlResolver/api-extractor/api-extractor.legacy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Most of the new `api-extractor.legacy.json` files added here appear to be unused. I don't think you need them.